.buildkite/pipeline.yml

```diff
@@ -6,7 +6,7 @@ agents:
 env:
   OPENBLAS_NUM_THREADS: 1
   JULIA_MAX_NUM_PRECOMPILE_FILES: 100
-  JULIA_DEPOT_PATH: "${BUILDKITE_BUILD_PATH}/${BUILDKITE_PIPELINE_SLUG}/depot/default"
+  #JULIA_DEPOT_PATH: "${BUILDKITE_BUILD_PATH}/${BUILDKITE_PIPELINE_SLUG}/depot/default"
```

Comment:
@haakon-e , @nefrathenrici  - I'm temporarily commenting out the depo. Could we add a script that clears it, the same as we do for other packages?